### PR TITLE
Cherry pick from Form branch

### PR DESCRIPTION
**Pull request by dustxd**

### CODE REVIEW

File: src/components/authentication/LoginPage.jsx [hunk 1]
```diff
@@ -71,18 +74,17 @@ class LoginPage extends Component {
...
-        </button>
         <button className={`${classes.button} ${classes.microsoft}`}>
           <img src={microsoft} className={classes.logo} alt={'Microsoft Logo'}></img>
           <div className={classes.divider}></div>
           <p className={classes.text}>Sign in with Microsoft Account</p>
         </button>
-        {/* <p className={classes.help}>Trouble signing in?<span className={classes.highlight}>Recover account</span></p> */}
-        <div className={classes.footer}><p className={classes.license}>© 2019 ACL Services Ltd. dba Galvanize. All Rights Reserved.</p></div>
+        <Typography align="center">or</Typography>
+        <button className={`${classes.button} ${classes.galvanize}`}>
```
Comment (yehee) [inline src/components/authentication/LoginPage.jsx]: I think I changed this to using `clsx` at some point, I can make that changes if you want me to!

File: src/components/calendar/CalendarPage.jsx [hunk 2]
```diff
@@ -70,11 +102,13 @@ const styles = theme => ({
     justifyContent: 'space-between',
   },
   button: {
-    border: 'none',
+    // border: 'none',
```
Comment (yehee) [inline src/components/calendar/CalendarPage.jsx]: It's safe to remove this line entirely!

File: src/components/authentication/LoginPage.jsx (hunk 1 above)
Comment (dustxd) [reply src/components/authentication/LoginPage.jsx]: What is clsx for? Compiling multiple stylesheets?

File: src/components/calendar/CalendarPage.jsx (hunk 2 above)
Comment (dustxd) [reply src/components/calendar/CalendarPage.jsx]: Okieee I'll remove other comments as well :)

**Pull request merged**